### ISSUE TITLE
Add your Project to this repo 

### ISSUE THREAD

**Issue by Logan1x** [OpenSource-TechnoNJR/First-Project#1]

**Comment by Logan1x** [comment]
@anandwana001 Please add description of this issue.

**Comment by anandwana001** [comment]
We would like to invite all the Techno-NJR student to start to GitHub and Open Source Contribution. Add your projects to this repo in this format **"your_year/your_semester/your_branch/your_name/your_project_name"** and start contributing to projects to help each other.

**Comment by PerfidusPistor** [comment]
can i add my Smartthings (IoT) project?

**Comment by Logan1x** [comment]
@PerfidusPistor  are you from techno?

**Comment by anandwana001** [comment]
@PerfidusPistor Yes you can, we had a little change!! You have to specify your college or job and add your project under **"your_country/your_city/your_project_domain(IOT)/your_name"**
I hope @Logan1x will agree with me as we are supporting open source, not in our college only but other also.

**Comment by PerfidusPistor** [comment]
i have got a problem to push the project. 
![image](https://user-images.githubusercontent.com/32550526/32136940-a11bcd7c-bc17-11e7-8d7b-08b6066b1f00.png)

**Comment by Logan1x** [comment]
Did  you forked the repo first? if not please fork it form top right and then clone it and add your changes.

**Comment by PerfidusPistor** [comment]
yes, i did. then i have added my project and commited, i have just troubles with "push" step

**Comment by Logan1x** [comment]
CC : @anandwana001
